scopes/lanes/ui/navigation/lane-switcher/lane-switcher.module.scss

```diff
@@ -13,6 +13,8 @@
 .laneGalleryIcon {
```

Comment:
can be removed from this PR 
Will be fixed in #6714 

https://github.com/teambit/bit/pull/6714/files#diff-0ec3f7734dff24c4d9ce2b21690bea773a5543a36dc7f3786025c98424e7dc6e